google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/SelectConfigDestinationFolderDialog.java

```diff
@@ -53,6 +55,17 @@ public SelectConfigDestinationFolderDialog(@Nullable Project project) {
         project,
         FileChooserDescriptorFactory.createSingleFolderDescriptor()
     );
+
+    // Present a canonical target folder as default in the path field.
+    if (project != null && project.getBasePath() != null) {
+      if (fileType == ConfigFileType.APP_YAML) {
+        destinationFolderChooser.setText(project.getBasePath() + "/src/main/appengine");
```

Comment:
One possible future enhancement would be to detect maven/gradle facets already attached to the project and default the directories based on that. If no information can be learned about the project then we default to the root of the project.